### ISSUE TITLE
 Features - Missing alt text 

### ISSUE THREAD

**Issue by SLovelace** [wet-boew/GCWeb#581]
WET Version: 4.0
Browser: Chrome 33.0.1750.146 m
Page affected: http://canada.ca/en/index.html
_(previously posted to Core Repo issues (ref #4791), closed and resubmitted here)_
ISSUE: "Features" missing alt text

**
Hello. Just wanted to send over a quick note about missing alt text on the "features" on the bottom of the Canada.ca home page. Looks like they don't have any Alt text.

Background:
I was running some WAVE tests on some pages for something else that turned up an error on the linked GOC FIP, so I thought I'd check out the homepage to see if the same error presented and Wave found 5 errors on the homepage including the features missing alts.
http://wave.webaim.org/report#/http://canada.ca/en/index.html

Not sure if this was maybe just a mistake, I also took a look at the elements individually (yep! Still missing Alts) and then just to be sure I ran a NVDA test to see how the links were read. NVDA just picked up "Link", "Link", "Link".

Anyway, just wanted to let you know and hope this helps. Let me know if you need more info, if you already know about it or you have questions.

thanks!
